installation/manual-installation/migrating-meteor-mongo/README.md

```diff
@@ -0,0 +1,19 @@
+# Migrating database from Meteor built in MongoDB
+
+This is a guide to migrate a manual installation database that is using the mongo database that comes built in meteor to an external mongo data base.
+
+We advise against using meteor's built in mongo on a production environment due performance reasons.
+
+Before starting make sure meteor is running.
+
+Then run: `mongodump -h 127.0.0.1 --port 3001 -d meteor`
+
+It'll dump the database in a folder called dump.
+
+Then install mongodb on your system. Please see the [mongo section](../ubuntu/) in one of our guides
```

Comment:
might link a bit higher out to any of the manual installs.  Cause could be on centos